MUI_Config/config.lua

```diff
@@ -604,17 +604,17 @@ function config:init()
                                     return current_value or "disabled";
                                 end,
                                 options = {
-                                    L["Combat_timer"],
```

Comment:
All looks good, why was the localization removed from this? Just curious. Thanks for the help